rollup.config.js

```diff
@@ -5,19 +5,22 @@ import pkg from "./package.json";
 
 export default [
   // browser-friendly UMD build
-  {
-    input: "src/index.ts",
-    output: {
-      name: "cms-bb2",
-      file: pkg.browser,
-      format: "umd",
-    },
-    plugins: [
-      resolve(),
-      commonjs(),
-      typescript({ tsconfig: "./tsconfig.json" }),
-    ],
-  },
+//   {
```

Comment:
Why is all of this being commented out?

Reply:
due to the 'express' thing, will re-surrect after re-factor out 'express'